pkg/context/httpcontext.go

```diff
@@ -105,6 +106,8 @@ type (
 		Std() *http.Request
 
 		Size() uint64 // bytes
+
+		TLS() *tls.ConnectionState
```

Comment:
propose not adding this function to keep the interface as minimal as possible, as the information can be retrieved from `Std().TLS`.
I and @suchen-sci are working on the design of the new pipeline, though not finished yet, we believe a more generic `Context` interface requires fewer code changes to implement the new design.

Reply:
https://github.com/megaease/easegress/pull/474/commits/b3d6f608aa880d0fc63a0f3032560b69c7bd6df4 fixes this